README.rst

```diff
@@ -460,6 +460,34 @@ will print
 Known Issues
 ------------
 
+InputExtensionNotFound or UnsupportedMimeType
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The current mimetype/file-extension database of preview-generator may differ between computer which
+mean you can get trouble to get preview depending on your OS or you installed software.
+
+In case you get one of theses exception and the mimetype/extension you tried is marked as
+supported by preview generator
+you should:
+- verify the version of PG you are using.
+- verify you have the proper dependencies to make you're builder work as expected.
+- check if the mimetype of the format you are using is handled by preview_generator mimetype_storage (python console):
```

Comment:
```suggestion
The current mimetype/file-extension database of preview-generator may differ between systems which
means you can struggle to get previews depending on your OS or you installed software.

In case you get one of these exceptions and the mimetype/extension you tried is marked as
supported by preview generator,  you should:
- check the version of PG you are using.
- check you have the proper dependencies to make you're builder work as expected.
- check if the mimetype of the format you are using is handled by preview_generator mimetype_storage (python console):
```